proparse/src/main/java/org/prorefactor/proparse/ParserSupport.java

```diff
@@ -287,28 +285,29 @@ SymbolScope parseSuper(JPNode classNode, String qualSuperName)
       throws IOException, ANTLRException {
     String superFileName = classFinder.findClassFile(qualSuperName);
     if (superFileName.length() == 0) {
-      // Could not find the super class. Will happen with Progress.lang.*,
-      // vendor libraries, etc.
+      // Could not find the super class. Will happen with Progress.lang.*, vendor libraries, etc.
       return null;
     }
 
-    try (BufferedReader ifstream = new BufferedReader(
-        new InputStreamReader(new FileInputStream(superFileName), session.getCharset()))) {
-      DoParse superDoParse = new DoParse(session, superFileName, /* FIXME doParse */ null);
-      superDoParse.inStream = ifstream;
-      superDoParse.doParse();
-      ParserSupport superSupport = superDoParse.getParserSupport();
-      if (!superSupport.isClass())
-        throw new ProparseRuntimeException(unitScope.getScopeName() + " inherits " + qualSuperName + " which is not a class.");
+    try {
+      ParseUnit unit = new ParseUnit(new File(superFileName), session);
+      unit.parse();
+
+      // FIXME Find if this really has to be kept 
+      // ParserSupport superSupport = parser.getParserSupport();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)